RELEASE-NOTES.md

```diff
@@ -1,5 +1,69 @@
 # Prefect Release Notes
 
+## Release 2.13.5
+
+### Load and serve remotely stored flows
+
+You can now load and serve flows from a git repository! 
+
+With the new `flow.from_source` method, you can specify a git repository and a path to a flow file in that repository. This method will return a flow object that can be run or served with `flow.serve()`.
+
+Here's an example of loading a flow from a git repository and serving it:
+
+```python
+from prefect import flow
+
+if __name__ == "__main__":
+    flow.from_source(
+        source="https://github.com/org/repo.git",
+        entrypoint="path/to/flow.py:my_flow",
+    ).serve(name="deployment-from-remote-flow")
+```
+
+When you load and serve a flow from a git repository, the serving process will periodically poll the repository for changes. This means that you can update the flow in the repository and the changes will be reflected in the served flow without restarting the serve script!
+
+To learn more about loading and serving flows from a git repository, check out our [the docs](https://docs.prefect.io/latest/concepts/flows/#retrieve-a-flow-from-remote-storage)!
+
+See the following pull request for details:
+- https://github.com/PrefectHQ/prefect/pull/10884
+
+### Experimental Pydantic 2 Compatibility
+
+We're working eagerly toward having `prefect` installable with either `pydantic<2` or `pydantic>2`.  As a first step toward compatibility, we've ensured that Prefect's use of `pydantic` is isolated from _your_ use of `pydantic` in as many ways as possible.  As of this release, `prefect` still has a stated `pydantic` requirement of `<2`, but we are testing against `pydantic>2` in our continuous integration tests.  If you're feeling adventurous, feel free to manually install `pydantic>2` and run some flows with it.  If you do, please let us know how it's going with a note in Slack or with a Github issue.
+
```

Comment:
See the following pull requests for details
- https://github.com/PrefectHQ/prefect/pull/10860
- https://github.com/PrefectHQ/prefect/pull/10867
- https://github.com/PrefectHQ/prefect/pull/10868
- https://github.com/PrefectHQ/prefect/pull/10870
- https://github.com/PrefectHQ/prefect/pull/10873
- https://github.com/PrefectHQ/prefect/pull/10891